Sources/TuistGraph/DependenciesGraph/DependenciesGraphNode.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import TSCBasic
+
+// A node of the third party dependency graph.
+public enum DependenciesGraphNode: Hashable, Equatable, Codable {
```

Comment:
To be more in line with the graph used for the rest of the project, I'd maybe rename this to `ThirdPartyDependency`? Or something like that? `Node` suffix has been in use before the introduction of `ValueGraph`.

Reply:
It makes sense to me.